context/app/static/js/components/home/HuBMAPDatasetsChart/hooks.ts

```diff
@@ -0,0 +1,96 @@
+import { SearchRequest } from '@elastic/elasticsearch/lib/api/types';
+import useSearchData from 'js/hooks/useSearchData';
+import {
+  DatasetTypeMapQueryAggs,
+  datasetTypeMapQuery,
+  organTypesQuery,
+  type OrganTypesQueryAggs,
+  type QueryAggs,
+} from './queries';
+
+export type AggregatedDatum = {
+  organ: string | number; // Actually just a string, but the union with Record below is necessary for TS to infer the type correctly
+} & Record<string, number>;
+
+export type AggregatedData = AggregatedDatum[];
+
+// Categorizes the data by organ, showing a count for each paired key
```

Comment:
We could benefit from some tests for these utils?

Reply:
I'll add some 👍🏻